Library/Formula/vamp-plugin-sdk.rb

```diff
@@ -0,0 +1,37 @@
+
+class VampPluginSdk < Formula
+  homepage "http://www.vamp-plugins.org/"
+  url "https://sourceforge.net/projects/vamp/files/vamp-plugin-sdk/2.2.1/vamp-plugin-sdk-2.2.1.tar.gz"
+  sha256 "571481098270133d2b78c6a461b850e04a98ab38284227c4d8056385f6333c26"
+
+  depends_on "pkg-config" => :build
+  depends_on "libsndfile"
+
+  patch :p1, :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!